docker/docker-compose-test.yaml

```diff
@@ -1,30 +1,39 @@
-# docker-compose -f ./docker/docker-compose-test.yaml up --build
-# docker-compose -f ./docker/docker-compose-test.yaml down
+# docker-compose -f ./docker/docker-compose-test.yaml up --build --force-recreate --detach
+# docker-compose -f ./docker/docker-compose-test.yaml down --volumes
 
 version: "2.4"
-x-net: &net
-  networks:
-      - main
+
 services:
   db:
-    <<: *net
-    image: datajoint/mysql:5.7
-    container_name: workflow-miniscope-test-db
+    networks:
+      miniscope:
+    image: mysql:5-debian
```

Comment:
Let's try reverting back to `datajoint/mysql:5.7` and seeing if there are any issues.

Reply:
By changing back, with a file that was otherwise working, I again see a 'handshake' error. This happens even when hopping in to the container and running `dj.conn()`

```python
OperationalError: (2003, "Can't connect to MySQL server on 'db' ([SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:997))")
```

Reply:
This error may be related to https://stackoverflow.com/questions/71006708/getting-sslv3-alert-handshake-failure-when-trying-to-connect-to-imap